setup.cfg

```diff
@@ -16,7 +16,7 @@ ignore = E221,E226,E241,E242, E731, E741, W0105, W503, N803, N806
 [flake8]
 max-line-length = 120
 max-doc-length = 100
-ignore = E221,E226,E241,E242, E731, E741, W0105, W503, N803, N806
+ignore = E221,E226,E241,E242, E731, E741, F722, W0105, W503, N803, N806
```

Comment:
It would be nice to add a description above: `# F722 syntax error in forward annotation`.

Reply:
Can we defer this (and below docstring) into a separate PR from @sdrave so we can merge this PR here right now?

Reply:
I guess not